Recipes/Controller/MainViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  MainViewController.swift
+//  Recipes
+//
+//  Created by krikaz on 3/12/20.
+//  Copyright © 2020 Lambda Inc. All rights reserved.
+//
+
+import UIKit
+
+class MainViewController: UIViewController {
```

Comment:
You can put your class as `final` since there will be no subclass of it. It prevents from subclassing.